docs/updating_schemas.md

```diff
@@ -3,37 +3,40 @@ Making changes to schemas
 
 This document describes how to make changes to the JSON schemas ([JSONSchema], [JSON-LD]) and source code that describes the encoded metadata model. For overview of code organization see [overview.md](https://github.com/IGVF-DACC/igvfd/blob/dev/docs/overview.md).
 
+----
+
 Guide to where to edit Source Code
 ----------------
 
-* **src/igvfd** Directory - contains all the Python and Javascript code for front and backends
-    * **audit** - Contains Python scripts that check JSON objects' metadata stored in the schema
-    * **schemas** - JSON schemas ([JSONSchema], [JSON-LD]) describing allowed types and values for all metadata objects
-    * **schemas/changelogs** - Schema change logs for documenting different versions of schemas
-    * **schemas/mixins.json** - Common schema property definitions for use with multiple schemas
-    * **tests** - Unit and integration tests
-    * **tests/data/inserts** - The sample data that comes up in a basic local instance.
-    * **tests/fixtures/schemas** - The sample Python objects to use with unit tests.  These Python function names are used as parameters in unit tests.
-    * **types** -  Business logic for dispatching URLs and producing the correct JSON
-    * **upgrade** - Python instructions for upgrading old objects to match the new schema
-    * **loadxl.py** - Python script that defines the schema objects to load
-    * **schema_format.py** - The format checker for accessions
-
-
------
+* **src/igvfd** Directory - contains all the Python code and JSON schemas for adding new types to the igvfd backend.
+    * **/audits** - Contains Python scripts that are run post-submission to check JSON objects' metadata stored in the schema
+    * **/schemas** - JSON schemas ([JSONSchema], [JSON-LD]) describing allowed types and values for all metadata objects
+      * **/schemas/changelogs** - Schema change logs for documenting the changes made in each version of a schema
+      * **/schemas/mixins.json** - Common schema property definitions for use with multiple schemas
+    * **/tests** - Unit and integration tests
+      * **/tests/data/inserts** - The sample data that comes up in a basic local instance.
+      * **/tests/fixtures/schemas** - The sample Python objects to use with unit tests.  These Python function names are used as parameters in unit tests.
+    * **/types** - Logic for dispatching URLs and producing the correct JSON.  This is where calculated properties are specified.
```

Comment:
Double space

Reply:
@jenjou Are the double spaces that you are finding the ones after the end of a sentence?  Or are are there mid-sentence ones?  Is the IGVF text format to drop the spacing after the period?

Reply:
I think I only saw them between (after the end) of a sentence. I'm used to having a single space -- I guess double space works, but we seem to have more single than double spaces so I thought it should be consistent.

Reply:
I see two spaces between sentences here. Markdown-display packages always strip spaces between sentences, so you can put in 1000 spaces after a period and Markdown strips it down to one. But I agree with Jennifer that keeping it consistent in the source as one space between sentences is nice. You’ll find decades of civil wars fought over whether one space or two between sentences was correct. I’m on the [one-space](https://www.cultofpedagogy.com/two-spaces-after-period/) battalion.